akka-actor/src/main/scala/akka/actor/Extension.scala

```diff
@@ -77,6 +77,11 @@ trait ExtensionId[T <: Extension] {
     java.util.Objects.requireNonNull(system, "system must not be null!").registerExtension(this)
   }
 
+  /**
+   * Returns an instance of the extension identified by this ExtensionId instance.
+   */
+  def apply(system: ClassicActorSystemProvider): T = apply(system.classicSystem)
```

Comment:
So the difference from what I tried in https://github.com/akka/akka/pull/27760#discussion_r326149501 is that it's defined in the `ExtensionId` instead of the concrete class. Great.